ts/components/messages/MessageList.tsx

```diff
@@ -195,6 +196,12 @@ class MessageList extends React.Component<Props, State> {
     }
   };
 
+  private scrollToIndex = (index: number) => {
```

Comment:
you could add _animated_ as a parameter and avoid to duplicate code (_scrollToTop_ function shares almost the same code)

Reply:
Done in commit 3cbad4b.